imports/ui/pages/professor/questions_public.jsx

```diff
@@ -154,18 +174,41 @@ class _QuestionsPublic extends Component {
 
 export const QuestionsPublic = createContainer(() => {
   const handle = Meteor.subscribe('questions.public')
-  let params = {
-    query: {
-      public: true
-    },
-    options: {
-      sort: { createdAt: -1 },
-      limit: 11
+  const user = Meteor.user()
```

Comment:
If I understand, the goal here is to limit the query for students to only show public questions that have a courseID for which the student is registered. In that case, we should instead change the 'questions.public' publication in questions.js to publish different results, and leave the code here as the same. I would recommend that it publish all questions for which the user is either in course.students or course.instructors (which I think is likely the same as checking that courseID is in user.profile.courses - is that true for an instructor?)

Reply:
Would this require the refactor of page_container so that all of the public questions a student can see are in the course they are currently within?